test/generated/testproto/grpc/dummy_pb2_grpc.pyi

```diff
@@ -8,61 +8,61 @@ import testproto.grpc.dummy_pb2
 import typing
 
 class DummyServiceStub:
-    """DummyService"""
+    r"""DummyService"""
     def __init__(self, channel: grpc.Channel) -> None: ...
     UnaryUnary: grpc.UnaryUnaryMultiCallable[
         testproto.grpc.dummy_pb2.DummyRequest,
         testproto.grpc.dummy_pb2.DummyReply] = ...
-    """UnaryUnary"""
+    r"""UnaryUnary"""
```

Comment:
Not a docstring, can we make it use the real comment syntax instead, which we had in the prior release?